concourse/scripts/concourse-lite-environment.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
If we add a new file `concourse-lite-environment.sh`, why do we need `environment.sh` at all in paas-bootstrap.

Anyway, I am not 100% confortable that both scripts are different